### DISCUSSION TITLE
Can the Action badge be altered?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#27004] [Actions]
<p>I want to use the GitHub Action badge to display the current status of a deployed page, but I would like to alter the text displayed to something different. For example: Instead of “passing” I would like it to display “page built”</p>
<p>Is there a way to alter the displayed text in such a way?</p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Hi <a class="mention" href="/andre601">@andre601</a>,</p>
<p>Looked into this for you. The badge text is not configurable. </p>
